docs/dataset_fields_for_iris_timestamp.md

```diff
@@ -0,0 +1,112 @@
+# Dataset Fields for IRIS Timestamp
+
+This document maps timestamp field names to their corresponding datasets.
```

Comment:
I think we should try to make this clearer, maybe something like the below: 

Below is a list of all the datasets in the IRIS Archive. 
 
When using the 'Dataset' tag filter, set the value of 'Dataset' to the desired acronym from the list below.

The Datasets are grouped by the reference of their 'Timestamp' tag (e.g BOALF's Timestamp tag references acceptanceTime, whilst BOAV's Timestamp references createdDateTime). Note that the tag is always called 'Timestamp', regardless of the time it references.